backends/cortex_m/ops/op_add_tensor.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <executorch/runtime/kernel/kernel_includes.h>
+
+extern "C" {
+#include "Include/arm_nnfunctions.h"
+}
```

Comment:
move this to a common include for `cortex_m/ops`? So we can do guards and check in one place.